python/ml_croissant/ml_croissant/_src/operation_graph/graph.py

```diff
@@ -19,7 +19,8 @@
     GroupRecordSet,
     InitOperation,
     Join,
-    ReadCsv,
+    ParseJson,
+    Read,
```

Comment:
Can you remind me why we prefer `read` vs. `readCsv`?

Reply:
Because now it reads both CSV and JSON. Tomorrow, it should read any file (JSONL, Parquet, etc). See also https://github.com/mlcommons/croissant/issues/79.